atcoder-problems-backend/src/bin/dump_json.rs

```diff
@@ -24,15 +25,27 @@ async fn main() -> Result<()> {
 
     let client = s3::S3Client::new()?;
 
-    let mut contests = pg_pool.load_contests().await?;
+    let mut contests = pg_pool
+        .load_contests()
+        .await?
+        .into_iter()
+        .filter(|c| !BLOCKED_CONTESTS.contains(&(&c.id as &str)))
```

Comment:
```suggestion
        .filter(|c| !BLOCKED_CONTESTS.contains(&(c.id.as_str())))
```
はどうでしょう？あまり as を多用したくない気持ちがあります。